scripts/conversion_script.py

```diff
@@ -428,6 +442,12 @@ def update_insights_inventory():
 
 def main():
     """Main entrypoint for the script."""
+    if os.path.exists(C2R_REPORT_FILE):
+        archive_analysis_report(C2R_REPORT_FILE)
+
+    if os.path.exists(C2R_REPORT_TXT_FILE):
+        archive_analysis_report(C2R_REPORT_TXT_FILE)
```

Comment:
I am just thinking what happens when some exception occurs in these calls, lets say report exists but somehow the archive function fails, the move not successful or the archive folder can't be created, would it be worth it to wrap it in try block? and do something based on expectations, do we expect that if this fails the script should fail and report the issue?

Maybe it's not necessary, if it's run under root it should work